src/gei/CommandArgs/GenerateScriptCommandArgs.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.IO;
+using OctoshiftCLI.Extensions;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter.CommandArgs;
+
+public class GenerateScriptCommandArgs : BaseCommandArgs
+{
+    [LogName("GITHUB SOURCE ORG")]
```

Comment:
I think for almost all of the. properties we won't need the `LogName` attribute, we can simply write a code to automatically covert the property name to a log friendly name. All that method needs to do is to split the property name by capital letters (except the very first one) then join them with a space and `ToUpper()` the end result.

Reply:
done